web/groups.go

```diff
@@ -130,6 +130,7 @@ func (s *AutograderService) updateGroup(ctx context.Context, sc scm.SCM, request
 	if err != nil {
 		return err
 	}
+	request.Users = users
```

Comment:
I'm uncomfortable to touch the request object in this way. Is this necessary?

Reply:
I tried to comment this line and the tests still pass. So we should augment the test with a scenario that capture the effect of this line.

Reply:
Yeah, the problem with scm-related tests is that they rely on `Fake` scm client, which is lagging behind a lot with respect to GitHub, so the test will pass here, but it still would not work in the real life. I've updated a few minor things, but maybe it is worth it to keep it more up to date.

Reply:
Ah, good point about the `Fake` scm client. Can we update this so that it becomes possible to test these particular things that we are interested in here...? Another option is to actually run tests against GitHub, using a similar approach to that in `web/hooks/github_test.go` and other places where we use `GITHUB_ACCESS_TOKEN`. Although the latter is a bit unsatisfying, since you expose your tests to non-determinism of GitHub access and behavior. It would be better to have repeatable test cases using the `Fake` client, that can also be run by the Travis CI...

Reply:
Yeah, added a task to the backlog, feels like it deserves a separate PR.